mite_websocket/__init__.py

```diff
@@ -15,22 +15,29 @@ class WebsocketError(MiteError):
 
 class _WebsocketWrapper:
     def __init__(self):
-        self.connection = None
+        self.connections = []
 
     def install(self, context):
         context.websocket = self
 
     async def uninstall(self, context):
-        if self.connection:
-            await self.connection.close()
+        for conn in self.connections:
+            if conn:
+                await self.close_connection(conn)
```

Comment:
Is it safe to call close on a connection that is already closed?  Or does the `if conn` test for closedness (which would be slightly unusual, but maybe that's how the websocket library works...?)

Reply:
I tried to close the same connection multiple times and no error are showed. There are errors if I try to send or recv on a closed connection but not error is showed to me closing multiple times.

Reply:
Thanks for checking, this code is OK then 🙂